Proposed/update_units.md

```diff
@@ -0,0 +1,25 @@
+#### **Title**
+Update `Units` Definitions with units in use at Strateos
+
+#### **Authorship**
+Yang Choo yang.choo@strateos.com
+
+#### **Motivation**
+There are a number of units we have since adopted at Strateos for chemistry and biology purposes which are currently not present in the Autoprotocol specification.
+
+#### **Proposal**
+Add the following units
+- Acceleration: `millimeter/second^2`
+- ElectricPotential: `nanovolt`, `microvolt`
+- Frequency: `kilohertz`, `rpm`
+- Length: `nanometer`
+- Matter: `nanomole`
```

Comment:
this definitely makes sense given how we currently specify units in the spec but at some point, we might want to consider handling metric prefixes independent from the units of measure themselves so that with a `mole` unit, `nanomole`, `micromole`, etc. are all implied.

side note: would ^ imply that we need to support picodays? I guess we could just specify that this only applies to SI base units?

Reply:
Speaking of SI units... not sure if this is the right place for this, but would it be possible to organize units in terms of `base` unit types (e.g. gram, meter, mole, watt, etc), and `prefixes` (e.g. mega (e^6), kilo (e^3), micro(e^-6), pico(e^-12), etc)? Feels like listing "micro...", "milli..." for each unit type all the time is cumbersome, and not well standardized... There need to be some exception for units like hour, rpm, etc, but it might be good to have defined `prefixes`?

Reply:
Oh yea, for sure :) 

I think we should definitely handle this better in the long-term, punting on this as a separate task for now.